code/modules/surgery/organs/augments_eyes.dm

```diff
@@ -8,6 +8,8 @@
 	w_class = 1
 
 	var/vision_flags = 0
+	var/darkness_view = 0
+	var/see_darkness = 1
```

Comment:
@Fox-McCloud i need help setting it up, or at least understanding how the night vision huds do it.